app/src/static/css/quiz.css

```diff
@@ -0,0 +1,271 @@
+    --input-border: 0.5px black solid;
+    --error-border: 0.4px red solid;
+
+    /* Transitions */
+    --link-transition: all 0.2s ease;
+    --underline-transition: transform 0.3s ease-out;
+    --btn-transition-1: all 0.1s linear;
+    --btn-transition-2: all 0.7s ease;
+}
+
+/**
+ * ========
+ * General Styling
+ * ========
+**/
+
+.body-text {
+    font: var(--body-font);
+}
+
+main a {
+    position: relative;
+}
+
+main a::after {
+    background-color: var(--textbox-bg-clr);
+    position: absolute;
+    content: "";
+    height: 1.5px;
+    width: 100%;
+    bottom: -2px;
+    left: 0;
+
+    transform: scaleX(0);
+    transform-origin: bottom left;
+    transition: var(--underline-transition);
+}
+
+main a:hover::after {
+    transform: scaleX(1);
+    transform-origin: bottom left;
+}
+
+/**
+ * ========
+ * Specific Element Styling
+ * ========
+**/
+#heading {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    margin-top: 60px;
+    margin-bottom: 20px;
+}
+
+#heading h1 {
+    font: var(--heading-font);
+    color: var(--heading-clr);
+}
+
+#quiz-container {
+    max-width: 1300px;
+    margin: 0 auto;
+    padding: 20px;
+    background-color: var(--box-clr);
+}
+
+#quiz-heading {
+    display: flex;
+    align-items: center;
+}
+
+#progress-container {
+    border: 1px solid var(--textbox-bg-clr);
+    background-color: var(--textbox-bg-clr);
+}
+
+.count-box {
+    color: var(--textbox-txt-clr);
+    padding: 8px;
+    margin: 0;
+    display: absolute;
+}
+
+#progress-bar {
+    margin: 8px;
+    height: 20px;
+    width: 100px;
+    color: var(--heading-clr);
+    border: 2px solid black;
+    border-radius: 3px;
+}
+
+progress::-webkit-progress-value {
+    background: var(--heading-clr);
+}
+
+progress::-webkit-progress-bar {
+    background: var(--checkbox-clr);
+}
+
+#questions-title {
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    margin-bottom: 20px;
+    color: var(--heading-clr);
+}
+
+#quiz-questions {
+    margin-top: 10px;
+}
+
+.question-block {
+    display: flex;
+    margin-bottom: 20px;
+    padding: 15px;
+    border: 1px solid var(--input-border);
+    border-radius: 5px;
+    background-color: var(--textbox-bg-clr);
+    flex-direction: column;
+}
+
+.question {
+    margin-right: 10px;
+    font-weight: bold;
+}
+
+/* Buttons */
+.question-options {
+    display: block;
+    margin-left: 20px;
+}
+
+.question-options label {
+    display: block;
+    margin-bottom: 10px;
+}
+
+label {
+    display: block;
+    margin-bottom: 10px;
+}
+
+input[type="radio"] {
+    margin-right: 8px;
+}
+
+#quiz-form button {
+    width: 100%;
+    height: 38.4px;
+    margin-top: 20px;
+    overflow: hidden;
+    position: relative;
+    transition: var(--btn-transition-1);
+    border: var(--btn-disabled-border);
+    color: var(--btn-txt-clr);
+    background-color: var(--btn-bg-disabled-clr);
+    box-shadow: none;
+}
+
```

Comment:
Add this here:

```python
#quiz-form button.clickable {
    border: none;
    color: var(--btn-txt-clr);
    background-color: var(--btn-bg-clr);
    box-shadow: var(--btn-shadow-2);
}
```
This is supposed to visibly change the button in its neutral state, showing the user that the submit button can now be interacted with. Without this, the button appears the same in both its disabled and enabled form, so the user can't easily tell whenever they're permitted to submit their answers (unless they randomly decide to click the button, which isn't very intuitive design).